website/versioned_docs/version-5.x/api.md

```diff
@@ -73,7 +73,7 @@ singleSpa.registerApplication({
 			<dt>'/app1'</dt>
 			<dd>✅ https://app.com/app1</dd>
 			<dd>✅ https://app.com/app1/anything/everything</dd>
-      <dd>🚫 https://app.com/app2</dd>
+			<dd>🚫 https://app.com/app2</dd>
```

Comment:
I prefer spaces usually nowadays, but am using tabs in this file to avoid merge conflict with pt-pt.single-spa.js.org and zh-hans.single-spa.js.org